driver-core/src/main/java/com/datastax/driver/core/DataTypeCqlNameParser.java

```diff
@@ -65,6 +65,8 @@
                     .put("timeuuid", timeuuid())
                     .put("tinyint", tinyint())
                     .put("smallint", smallint())
+                    // duration is not really a native CQL type, but appears as so in system tables
+                    .put("duration", duration())
```

Comment:
It turns out it will be in the system tables in C* 4.x (v5 protocol), but is a custom type in v4 protocol.